### ISSUE TITLE
Provide .rpm and .deb installers on the lokinet website

### ISSUE THREAD

**Issue by Destroyinator69420** [oxen-io/lokinet#2151]
I have been using Lokinet for a while now. I am concerned about the lackluster support for RPM based distributions and the inability to easily get binaries outside of a package repository. I use package repositories for most of my apps, but no Lokinet, I2P or Tor. I do not want my computer giving away the fact that I use these technologies to every WIFI network I connect to and I hope you understand this. I want there to be a Ubuntu .deb targeted at the latest Ubuntu stable, right now its 22.10 and an RPM targeted at the latest Fedora stable, right now its version 37. I do not want to use package repositories for anything cryptocurrency/darknet/anonymity related since that would give away the fact that I use these technologies to every network I connect to. It should be one file that includes Lokinet, Lokinet VPN and the Lokinet GUI in one package for ease of install. On the Lokinet website fedora and Ubuntu downloads would be differentiated with a fedora logo and a Ubuntu logo respectively. This is the main thing stopping people from adopting Lokinet is an easy to use alternative to Tor.

**Comment by majestrate** [comment]
we distribute the debs via the apt repository for a reason, that is how debian packaging does signing, the whole apt repo is signed.

on RPM, we have yet to put that into our main CI pipline, see https://github.com/oxen-io/lokinet/issues/1979 .

closing as duplicate of #2087

**Comment by jagerman** [comment]
You can also grab the debs out of the repository manually; for Ubuntu 22.10 you want the ones with `~ubuntu2210` in the package name.  I believe you'd need just these three for a working lokinet:
- `lokinet`
- `lokinet-bin`
- `liboxenmq1.2.14`
from https://deb.oxen.io/pool/main/l/lokinet/ and https://deb.oxen.io/pool/main/o/oxenmq/

The deb repository is also available over lokinet itself once you have lokinet running, at http://deb.loki so that you can keep a repository for easier updates, without revealing your address when fetching updates/package lists.

**Comment by majestrate** [comment]
note: we also have branches for each of the supported packing targets that people can build the deb/rpm themselves from source. e.g: 

* https://github.com/oxen-io/lokinet/tree/debian/bookworm 

* https://github.com/oxen-io/lokinet/tree/ubuntu/kinetic 

* https://github.com/oxen-io/lokinet/tree/fedora/37

**Comment by necro-nemesis** [comment]
RPMs for Fedora including F37 have been built by CI for a considerable time just not in the main pipeline. Electron GUI was also not and possibly still isn't in the repo although at one point there was a F37 cli build there. The rpm file was uploaded by CI to the CI build repo when my PRs were merged and not to the rpm.oxen.io repo auto-magically. Previously taking this build and distributing it was a manual process. I only merged and PR'd when a new release was made available so likely what is there likely dates back to whatever the last release was. At some point my folder was unceremoniously deleted so without checking I can't confirm a file is there. I do recall merging awhile back just to regenerate the deleted file.
